tests/clang_parser.cpp

```diff
@@ -30,16 +30,17 @@ static void parse(const std::string &input,
 TEST(clang_parser, integers)
 {
   BPFtrace bpftrace;
-  parse("struct Foo { int x; int y, z; }", bpftrace);
+  parse("struct Foo { int x; int y, z; _Bool b; }", bpftrace);
```

Comment:
does regular `bool` work? would be good to test that. not sure if libbpf generates `_Bool` or `bool`

Reply:
Interestingly it doesn't here in the tests unless I explicitly include `stdbool.h` (which I did). When running bpftrace from CLI, using `bool` works without the include (as `ClangParser` includes `stdbool.h` implicitly). I don't understand why that doesn't work in the tests, though.